src/pudl/extract/ferc1.py

```diff
@@ -685,7 +685,7 @@ def get_ferc1_meta(ferc1_engine):
 
 
 def extract(
-    ferc1_settings=Ferc1Settings(),
+    ferc1_settings: Ferc1Settings = Ferc1Settings(),
```

Comment:
Huh, so for the `epacems` and `ferc1` we were already using the Settings objects?

Reply:
I think I did this during the metadata refactor, but left out the EIA sources because they used the generic extractor.